tools/pkgutils.py

```diff
@@ -153,7 +156,7 @@ def _git_describe(is_exact, git_dir, cwd):
 # git describe return "0.1-143-ga554734"
 # git_describe() returns {'release': '143', 'tag': '0.1', 'hash': 'ga554734'}
 def git_describe(is_exact=False, split=True, cwd=None):
-
+    """Running git describe command """
```

Comment:
There's valuable information in comments right above this function that belongs in the docstring.